website/docs/concepts/router.mdx

```diff
@@ -45,7 +45,9 @@ matched, the router navigates to the path with `not_found` attribute. If no rout
 a message is logged to console stating that no route was matched.
 
 Finally, you need to register one of the Router context provider components like `<BrowserRouter />`.
```

Comment:
I don't think the phrasing to "register" a context is used elsewhere, and personally I find it confusing. One phrasing that comes to mind is

> The router components, e.g. `<Link />` and `<Switch />`, must be used in the [context](concepts/contexts.mdx) of a Router provider like `<BrowserRouter />`.

Then you can also get rid of the "I.e. certain components [..]" sentence later on.

Reply:
Ah ok I really like this. I actually had no clue that the Router was a context. This is good to know. I think that would help clear things up.